tests_metricflow/snapshots/test_metric_filter_rendering.py/SqlQueryPlan/DuckDB/test_multi_hop_inner_join_path__plan0_optimized.sql

```diff
@@ -0,0 +1,42 @@
+-- Constrain Output with WHERE
```

Comment:
I'm a little confused about where these snapshots are coming from. Some other PR?

Reply:
Dang it, I had a test I was running in this PR and ended up deleting it (but forgot to delete the snapshots). Will remove!